lldb/include/lldb/API/SBDebugger.h

```diff
@@ -153,70 +249,221 @@ class LLDB_API SBDebugger {
   /// lldb::SBStructuredData settings = debugger.GetSetting("target.arg0");
   /// lldb::SBStructuredData settings = debugger.GetSetting("target");
   ///
-  /// \param[out] setting
+  /// \param[in] setting
   ///   Property setting path to retrieve values. e.g "target.source-map"
   ///
+  /// \return
+  ///   An SBStructuredData object containing the requested settings.
   lldb::SBStructuredData GetSetting(const char *setting = nullptr);
 
+  /// Set whether the debugger should run in asynchronous mode.
+  ///
+  /// When in asynchronous mode, events are processed on a background thread.
+  ///
+  /// \param [in] b
+  ///   True to enable asynchronous mode, false for synchronous mode.
   void SetAsync(bool b);
 
+  /// Get whether the debugger is running in asynchronous mode.
+  ///
+  /// \return
+  ///   True if the debugger is in asynchronous mode, false otherwise.
   bool GetAsync();
 
+  /// Set whether to skip loading .lldbinit files.
+  ///
+  /// \param [in] b
+  ///   True to skip loading LLDB init files, false to load them.
   void SkipLLDBInitFiles(bool b);
 
+  /// Set whether to skip loading application-specific .lldbinit files.
```

Comment:
Is this one specifically for Xcode ? I don't think the documentation makes it clear if that's so.

Reply:
No, this uses `argv[0]` and works with any tool.